gomint-server/src/main/java/io/gomint/server/command/internal/GamemodeCommand.java

```diff
@@ -0,0 +1,69 @@
+package io.gomint.server.command.internal;
+
+import io.gomint.ChatColor;
+import io.gomint.GoMint;
+import io.gomint.command.Command;
+import io.gomint.command.CommandOutput;
+import io.gomint.command.annotation.*;
+import io.gomint.command.validator.IntegerValidator;
+import io.gomint.command.validator.StringValidator;
+import io.gomint.command.validator.TargetValidator;
+import io.gomint.entity.EntityPlayer;
+import io.gomint.world.Gamemode;
+
+import java.util.Map;
+
+/**
+ * @author NycuRO
+ * @version 1.0
+ */
+@Name( "gamemode" )
+@Description( "Sets a player's game mode" )
+@Permission( "gomint.command.gamemode" )
+@Overload( {
+    @Parameter( name = "gameMode", validator = IntegerValidator.class),
+    @Parameter( name = "gameMode", validator = StringValidator.class, arguments = { "[a-zA-Z0-9ß\\-]+" })
+} )
+@Overload( {
```

Comment:
This overload just needs one target parameter, also wrong.
An overload is not equal to an argument.